data/saas/config/logiid_config.yml

```diff
@@ -0,0 +1,55 @@
+saas_config:
+    fides_key: logiid_connector_example
+    name: Logiid SaaS Config
```

Comment:
Let's use `Logi ID` whenever it's not used as an identifier, so for example the name and description